README.md

```diff
@@ -15,25 +15,135 @@ You can reach the maintainers of this project at:
 -->
 
 ------
+## Description
+
+### Work agent
+Work agent is a controller running on the managed cluster. It watches the `ManifestWorks` CRs in the cluster namespace on hub cluster and applies the manifests included in those CRs on the managed clusters.
```

Comment:
we need to define what is cluster namespace. Work agent does not require a cluster namespace notion here actually. You can say that work agent will monitor the work resource on a certain ns on hub, in which the work agent has permission.

Reply:
Doc updated.